release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -749,13 +749,18 @@ In the following tables, features are marked with the following statuses:
 |Admin Network Policy (`AdminNetworkPolicy`)
 |Technology Preview
 |Technology Preview
-|Technology Preview
+|General Availability
 
 |IPsec external traffic (north-south)
 |Technology Preview
 |General Availability
 |General Availability
 
+|A `MachineConfig` object with a customized `br-ex` bridge
```

Comment:
cc @qiowang721 and @cybertron

Reply:
Hi @JoeAldinger Thanks for your helping on the doc update.
I'm thinking about the feature name, the epic name is configure-ovs alternative, but seems it cannot indicate what the feature is explicitly...hmm, how about 'configure the br-ex bridge needed by OVNK using NMState'? @cybertron WDYT?

Reply:
@JoeAldinger  thanks for updating, LGTM.
Let's wait for comments from @cybertron .